examples/run/sycl/full_chain_algorithm.sycl

```diff
@@ -73,6 +75,7 @@ full_chain_algorithm::full_chain_algorithm(
       m_device_detector{},
       m_clusterization{memory_resource{m_cached_device_mr, &(m_host_mr.get())},
                        m_copy, m_data->m_queue_wrapper, clustering_config},
+      m_measurement_sorting(m_copy, m_data->m_queue_wrapper),
```

Comment:
clusterization does not guarantee the sorted measurement output? @stephenswat

Reply:
Not without #666.

Reply:
ok